tests/acceptance/new-test.js

```diff
@@ -65,6 +65,19 @@ describe('Acceptance: ember new', function() {
     confirmBlueprinted();
   }));
 
+  it('MODULE_UNIFICATION=true ember new foo works', co.wrap(function *() {
+    process.env.MODULE_UNIFICATION = 'true';
+    yield ember([
+      'new',
+      'foo',
+      '--skip-npm',
+      '--skip-bower',
+    ]);
+
+    confirmBlueprintedForDir('blueprints/module-unification-app');
+    process.env.MODULE_UNIFICATION = undefined;
```

Comment:
Should probably done in an `afterEach` in case the generation blows up and leaves us in a bad state...